### ISSUE TITLE
Create a prior knowledge assessment

### ISSUE THREAD

**Issue by moorepants** [moorepants/me41055#11]
It would be nice to have students fill out a survey before starting the course that answers questions about their prior knowledge and prerequisites. I may have one from from my old course (MAE 223) that can be adapted.

**Comment by moorepants** [comment]
We can likely use Brightspace quizzes for something like this, but we can also use google forms or something similar if we don't collect any student data that should be private. I have a number of these for prior courses using google forms.

**Comment by moorepants** [comment]
@zofiatycz I shared a google form with you in the tu delft google drive. I think we can use it almost as is, but maybe you want to review and see if I should add/subtract anything.

We then need to:

- Make an assignment 0 on brightspace
- Link to the google form in the assignement (only the public link for filling it out)
- Make the due date 17:00 Feb 4 (i'll ask them to fill it out in the first work session)

**Comment by zofiatycz** [comment]
I think that form can be used as-is, there is nothing I would subtract. I looked through the schedule and I think it covers most of basic topics. 
Shouldn't due date be 17:00 Feb11?

**Comment by moorepants** [comment]
>  Shouldn't due date be 17:00 Feb11?

Yes, my mistake above.

**Comment by moorepants** [comment]
@zofiatycz will make the due date 17:00 on February 8 (same day as first work session). Assignment created in brightspace which links to the form. Jason will take some time in the work session for people to fill it out.

**Comment by moorepants** [comment]
@zofiatycz Can you post this to the students on Monday morning?

**Comment by zofiatycz** [comment]
The assignment is created, not visible yet. If there is no comment I will make it visible on Monday morning (9 am)

**Comment by moorepants** [comment]
Sounds good!

**Comment by zofiatycz** [comment]
Done
